README.md

```diff
@@ -1,16 +1,31 @@
 
-## To run on Tallgrass or Yeti with Singularity:
-These are exact commands for Yeti, Tallgrass may require small changes.
+As of 2022, the repo is set-up for use on Tallgrass because Yeti will be removed from service in August 2022. The 2022 version of this repo is currently cloned here: `/caldera/projects/usgs/water/iidd/datasci/data-pulls/national-flow-observations`
+
+## To setup on Tallgrass with Singularity:
+These are exact commands for Tallgrass. This is needed only when the image needs to be updated or when we move the repo somewhere other than Tallgrass.
 
 After cloning the repo, pull the docker image from code.chs.usgs.gov:
 
 ```
-module load singularity/3.4.1
-singularity pull --docker-login docker://code.chs.usgs.gov:5001/wma/wp/national-data-pulls:v0.0
+module load singularity
+singularity pull --docker-login docker://code.chs.usgs.gov:5001/wma/wp/national-data-pulls:v0.1
 ```
+After you run the commands above you will see a prompt for a password. The password it is looking for is your [personal access token (PAT)](https://code.chs.usgs.gov/-/profile/personal_access_tokens). If you do not have a PAT set up you will need to do so.
+
+## To run on Tallgrass with Singularity:
+
+Two Slurm scripts are included in this repo.  If you want to run Rstudio to do development work, run `sbatch launch_rstudio.slurm` and then follow the instructions in the Slurm output file (`shellLog/rstudio.out`) to make an SSH tunnel and log in to Rstudio via a browser.
+
+To run the pipeline as a non-interactive batch job, open `run_scmake.slurm` and modify parameters like job length (`--time`) and partition (`-p`) as needed for how long you expect the job to run (see [this issue](https://github.com/USGS-R/national-flow-observations/issues/4) for estimates). Submit the job with `sbatch --mail-user=$USER@usgs.gov run_scmake.slurm`.
 
-Two Slurm scripts are included in this repo.  If you want to run Rstudio to do development work, run `sbatch launch_rstudio.slurm` and then follow the instructions in the Slurm output file to make an SSH tunnel and log in to Rstudio via a browser.
+To be able to run the full pipeline, you will need to have AWS credentials setup on your user's home directory on Tallgrass. You need to have the file `/home/username/.aws/credentials`, which looks like this:
+
+```
+[default]
+aws_access_key_id = ***
+aws_secret_access_key = ***
+```
 
-To run the pipline as a non-interactive batch job, open `run_scmake.slurm` and modify parameters like job length (`--time`) and partition (`-p`) as needed for how long you expect the job to run (see [this issue](https://github.com/USGS-R/national-flow-observations/issues/4) for estimates). Submit the job with `sbatch --mail-user=$USER@usgs.gov run_scmake.slurm`.
+The `aws_access_key_id` and `aws_secret_access_key` values are from the `dev-owi-s3-access` secret stored in `dssecrets`. Talk with your data science colleagues to get access to that secret if you don't already have it. 
```

Comment:
Would be good to point to `saml2aws` docs longer term for authentication.

Reply:
Yes, absolutely agree, David. I'll open an issue for it

Reply:
Yes, agree! Just was in a time crunch for this one. @padilla410 made note of that needed switch in #25

Reply:
jinx @padilla410